tools/oncotator/oncotator.xml

```diff
@@ -15,7 +15,7 @@
     </requirements>
     <command>
         source \${ONCOTATOR_PATH}/activate;
-        python \${ONCOTATOR_PATH}/oncotator
+        python oncotator.py
```

Comment:
@kgururaj Oncotator seems to be never ending. I just tried the tool at the end of the somatic workflow, and since we are in the job_working_directory during execution, oncotator.py is not being found. Is there an environment variable to point to Galaxy tools folder?